src/anomalib/data/__init__.py

```diff
@@ -25,6 +26,20 @@
 logger = logging.getLogger(__name__)
 
 
+class SupportedDataType(str, Enum):
```

Comment:
```suggestion
class DataFormat(str, Enum):
```

Reply:
the rest of anomalib refers to this as `format`, so it would be more consistent if this object is named as `DataFormat`

Reply:
all fixed.